### DISCUSSION TITLE
LDAP login issue

### DISCUSSION THREAD

**Discussion by nickdring** [joomla/joomla-cms#35312] [Q&A]
Hi there,
I migrated a 3.10 to the 4.0 that was finally released last week.
All good so far apart from an issue I'm having with LDAP login.
I'm using the standard LDAP plugin, there is no third party stuff on the site whatsoever.
On the same test site with J3 login via ldap worked fine but with the exact same settings new user and existing users are not able to log in.
The logs don't show anything but on the actual front end I see this error:
0 Could not set value "60" for option "network_timeout".
Attached is a screen shot showing the ldap settings.
https://www.dropbox.com/s/whnzuahaywwfqli/ldap.png

**Comment by PhilETaylor** [comment]
Please enable DEBUG mode (bottom option on that screen) and try again.

**Reply by nickdring** [reply]
HI Phil, I'm not seeing any difference, here is a screen shot.
https://www.dropbox.com/s/zth93wohxde9ncc/Screenshot.png

**Reply by PhilETaylor** [reply]
Now turn on Debug in Joomla Global Configuration and try again - this will give a huge stack trace - please screenshot that and post here: https://github.com/joomla/joomla-cms/issues/35316

**Comment by PhilETaylor** [comment]
Moved to https://github.com/joomla/joomla-cms/issues/35316 as a genuine issue to resolve.
